.github/release-drafter.yml

```diff
@@ -0,0 +1,45 @@
+# Config for https://github.com/release-drafter/release-drafter
+name-template: 'Akka Serverless Java SDK $NEXT_PATCH_VERSION'
+tag-template: 'v$NEXT_PATCH_VERSION'
+categories:
+  - title: '🚀 Productboard Features'
+    labels:
+      - 'Feature'
+  - title: ':toolbox: Chore'
+    labels:
+      - 'chore'
+  - title: 'Documentation'
+    labels:
+      - 'Documentation'
```

Comment:
How does it work for things marked with more than one label, for example Documentation and chore?

I'm thinking it might be better to base the categories solely on the conventional commit prefix rather than also looking for file locations.